sklearn/preprocessing/data.py

```diff
@@ -1708,49 +1716,56 @@ class OneHotEncoder(BaseEstimator, TransformerMixin):
 
         Non-categorical features are always stacked to the right of the matrix.
 
-    dtype : number type, default=np.float
+    dtype : number type, default=np.float64
         Desired dtype of output.
 
     sparse : boolean, default=True
         Will return sparse matrix if set True else will return an array.
 
     handle_unknown : str, 'error' or 'ignore'
-        Whether to raise an error or ignore if a unknown categorical feature is
-        present during transform.
+        Whether to raise an error or ignore if an unknown categorical
+        feature is present during transform.
 
     Attributes
     ----------
+    feature_index_range_ : array, shape [n_feature, 2]
+        ``feature_index_range_[i]`` specifies the range of column indices
+        occupied by the input feature `i` in the one-hot encoded array.
+
+    one_hot_feature_index_ : array, shape [n_features_new]
+        ``one_hot_feature_index_[i]`` specifies which feature of the input
+        is encoded by column `i` in the one-hot encoded array.
+
     active_features_ : array
         Indices for active features, meaning values that actually occur
         in the training set. Only available when n_values is ``'auto'``.
 
-    feature_indices_ : array of shape (n_features,)
-        Indices to feature ranges.
-        Feature ``i`` in the original data is mapped to features
-        from ``feature_indices_[i]`` to ``feature_indices_[i+1]``
-        (and then potentially masked by `active_features_` afterwards)
-
     n_values_ : array of shape (n_features,)
-        Maximum number of values per feature.
+        Number of categories per feature. Has value `0` for
```

Comment:
I changed to "Number of encoded categories" and moved `active_features_` back to deprecation. Hopefully that will make it clearer.